Lib/test/test_bytes.py

```diff
@@ -962,6 +963,45 @@ def test_translate(self):
         c = b.translate(None, delete=b'e')
         self.assertEqual(c, b'hllo')
 
+    def test_octal_values(self):
```

Comment:
It is not correct place for such test. This file is for tests of bytes and bytearray methods, not parsing Python code.

`test_string_literals` is for parsing string and bytes literals. Add some tests there.